### PR TITLE
December UI Release

### PR DESCRIPTION
**Pull request by Centerville1**
This release contains all the changes from the Lost and Found Senior Project, Where's My RA Senior Project, plus a few other minor improvements.

‼️ Wait to merge until API is merged and tested for stability.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -14,16 +14,12 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
```
Comment (Centerville1) [inline .github/workflows/ci.yml:17]: @EjPlatzer are these changes intentional, or is this something that needs to be reverted?  I believe these were your changes, but just wanted to check.

**Comment by EjPlatzer** [comment]
I'm marking this as a draft for now because we need to merge the API and make sure it's stable first.

File: .env.development [hunk 2]
```diff
@@ -7,4 +7,5 @@
  VITE_API_URL=https://360ApiTrain.gordon.edu/
 
 # @LOCALHOST
-# VITE_API_URL=http://localhost:51626/
+#  VITE_API_URL=http://localhost:51660/
```
Comment (Centerville1) [inline .env.development:10]: @EjPlatzer I'm guessing we need to revert this.

**Review by EjPlatzer — APPROVED** [review]
LGTM! I think this is ready to for a beta release, great work all!
